README.md

```diff
@@ -1,6 +1,7 @@
 # Presto
 [![Maven Central](https://img.shields.io/maven-central/v/io.prestosql/presto-server.svg?label=Download)](https://prestosql.io/download.html)
 [![Presto Slack](https://img.shields.io/static/v1?logo=slack&logoColor=959DA5&label=Slack&labelColor=333a41&message=join%20conversation&color=3AC358)](https://prestosql.io/slack.html)
+[![Presto: The Definitive Guide book download](https://img.shields.io/badge/Presto%3A%20The%20Definitive%20Guide-download-brightgreen)](https://prestosql.io/presto-the-definitive-guide.html)
```

Comment:
Unfortunately, that's not link to download the book. That's just a summary page.

Reply:
@martint oh snap. I’ll fix that